benefit/data/sheets/FixedLists/FixedLists.json

```diff
@@ -196,6 +196,7 @@
   {"LiveFrom": "01/01/2018", "ID": "FL_dwpStates", "ListElementCode": "adjournmentNoticeIssued", "ListElement": "Adjournment notice issued"},
   {"LiveFrom": "01/01/2018", "ID": "FL_dwpStates", "ListElementCode": "hearingRecordingProcessed", "ListElement": "Hearing recording processed"},
   {"LiveFrom": "01/01/2018", "ID": "FL_dwpStates", "ListElementCode": "hearingRecordingActioned", "ListElement": "Hearing recording actioned"},
+  {"LiveFrom": "12/08/2022", "ID": "FL_dwpStates", "ListElementCode": "hearingDateIssued", "ListElement": "Hearing Date Issued"},
```

Comment:
Don't add a field to both non-prod and prod sheets, it will cause an error. If you want something behind a feature flag and not to go to production, add it to a nonprod sheet.